src/applications/facility-locator/utils/helpers.jsx

```diff
@@ -14,6 +15,47 @@ export const setFocus = selector => {
   }
 };
 
+export const clearLocationMarkers = () => {
+  const locationMarkers = window.document.getElementsByClassName(
+    'mapboxgl-marker',
+  );
+  for (let i = locationMarkers.length - 1; i >= 0; i--) {
+    locationMarkers[i].remove();
+  }
```

Comment:
```suggestion
  Array.from(locationMarkers).forEach(marker => marker.remove());
```

Reply:
Note that `locationMarkers` is an `HTMLCollection` and it does not have the `forEach` method. But I still think this is slightly preferable to the C-style for loop.